box-ios-sdk-tests.podspec

```diff
@@ -0,0 +1,34 @@
+
+Pod::Spec.new do |s|
+
+  s.name         = "box-ios-sdk-tests"
+  s.version      = "1.0.0"
```

Comment:
Yes. Although we can leave the :tag blank to ensure it always pull the head, it does not referring to the latest release. Also it generates a pod spec lint warning.